frontends/main/src/app-pages/UnitsListingPage/UnitsListingPage.test.tsx

```diff
@@ -63,29 +90,29 @@ describe("DepartmentListingPage", () => {
 
     setMockResponse.get(
       urls.channels.counts("unit"),
-      units.map((unit) => {
+      unitChannels.map((channel) => {
         return {
-          name: unit.code,
+          name: channel.name,
           counts: {
-            courses: courseCounts[unit.code],
-            programs: programCounts[unit.code],
+            courses: courseCounts[channel.name],
+            programs: programCounts[channel.name],
           },
         }
       }),
     )
-    setMockResponse.get(urls.offerors.list(), {
-      count: units.length,
-      results: units,
+    setMockResponse.get(urls.channels.list({ channel_type: "unit" }), {
+      count: unitChannels.length,
+      results: unitChannels,
     })
 
-    units.forEach((unit) => {
-      setMockResponse.get(urls.channels.details("unit", unit.code), {
-        channel_url: `${window.location.origin}/units/${unit.code}`,
-      })
-    })
+    // units.forEach((unit) => {
```

Comment:
Now that this page is using the channel list endpoint to get the details, this is meant to be removed right?

Reply:
Yes, missed that - removed.